api/user/user.go

```diff
@@ -20,14 +21,42 @@ func NewUser(email, pass, keyID string) (*User, error) {
 	if err != nil {
 		return nil, fmt.Errorf("could not hash password: %s", err)
 	}
+
 	return &User{
 		Email:      email,
 		HashedPass: string(hash),
 		KeyID:      keyID,
+		Projects:   []string{},
 	}, nil
 }
 
 // CheckPassword verifies that a password matches the hash on the user
 func (u *User) CheckPassword(pw string) error {
 	return bcrypt.CompareHashAndPassword([]byte(u.HashedPass), []byte(pw))
 }
+
+// Adds a Project to the user
+func (u *User) AddProject(projectID string) {
```

Comment:
change the receiver of these to "name" and check elsewhere in the file we dont use 'id' to refer to the project's unique identifier